### PR TITLE
Fix an issue on AWS-GuardDuty content Pack when no findings are fetched

### PR DESCRIPTION
**Pull request by JeromeSola**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

Related to https://github.com/demisto/content/pull/31353

## Description
Fix an issue where if no findings are fetched, the last_updated_time is null and the dateparser.parse return an error
error_aws_guard_duty_content_pack
## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by shmuel44** [comment]
Hi @JeromeSola.
Thank you for your contribution.
Have you checked that your changes fixed the bug?
Why is the `last_updated_time` `null` in the first place?
How can we reproduce the bug?

Please feel free to reach out to me with any questions
I'm available here or on slack :)
Thanks again

**Comment by shmuel44** [comment]
Hi @JeromeSola, please complete the  [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)?
Thank you.

File: Packs/AWS-GuardDuty/ReleaseNotes/1_3_44.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### AWS - GuardDuty
+- Fix an issue on AWS-GuardDuty content Pack when no findings are fetched.
```
Comment (amshamah419) [inline Packs/AWS-GuardDuty/ReleaseNotes/1_3_44.md:3]:
```suggestion
- Fix an issue where the integration would fail when no findings are fetched.
```

**Review by amshamah419 — DISMISSED** [review]

**Comment by JeromeSola** [comment]
Hello ,
Thank you fr answering to my PR.
Sorry for the delay of answer on my side i was in hollydays.

Have you checked that your changes fixed the bug? Yes we currently use my update in our production xsoar 
Why is the last_updated_time null in the first place? Because the list of findings in Guard Duty for newly aws account is empty
How can we reproduce the bug? by having an aws account with an empty Guard Duty finding list

It's my first time trying to do a PR on your repo. 
If you could help me to make your CI/CD succeed or if you need more info on the issue thank you to let me know

have a good day
Jérome

**Comment by shmuel44** [comment]
Hi @JeromeSola,
the code looks good!

We're ready for a demo. 
Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR or send me a video of the demo.

**Comment by shmuel44** [comment]
Hello @JeromeSola,
Thank you for submitting your pull request.
Unfortunately, we will have to reject this PR at this time as we have already assigned one of our developers to work on the same issue.
We plan to merge the fix in the next one or two days and it will be available in 1.3.45 of the integration.
Contributions are welcome, and we're looking forward to seeing you commit here again. Please let me know if you have any further questions.

https://github.com/demisto/content/pull/32685

**Pull request closed without merging**